modules/offers/client/components/OfferLocation.component.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OfferLocationPresentational from './OfferLocationPresentational';
+
+export default class OfferLocation extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      zoom: 13,
+      windowWidth: 0
+    };
+
+    this.handleChangeZoom = this.handleChangeZoom.bind(this);
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+  }
+
+  /* updating windows size with https://stackoverflow.com/a/42141641 */
+  componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
```

Comment:
This is good here now, just noting for future situations where ResizeObserver might come handy:

https://developers.google.com/web/updates/2016/10/resizeobserver
https://www.npmjs.com/package/resize-observer-polyfill

Reply:
Oh right, actually not here because we're specifically interested in `windowWidth` only. :-)